span.go

```diff
@@ -135,15 +133,38 @@ type FinishOptions struct {
 	// (per StartSpanOptions).
 	FinishTime time.Time
 
-	// BulkLogData allows the caller to specify the contents of many Log()
+	// LogRecords allows the caller to specify the contents of many LogFields()
 	// calls with a single slice. May be nil.
 	//
-	// None of the LogData.Timestamp values may be .IsZero() (i.e., they must
+	// None of the LogRecord.Timestamp values may be .IsZero() (i.e., they must
 	// be set explicitly). Also, they must be >= the Span's start timestamp and
 	// <= the FinishTime (or time.Now() if FinishTime.IsZero()). Otherwise the
 	// behavior of FinishWithOptions() is undefined.
 	//
-	// If specified, the caller hands off ownership of BulkLogData at
+	// If specified, the caller hands off ownership of LogRecords at
 	// FinishWithOptions() invocation time.
+	//
+	// If specified, the (deprecated) BulkLogData must be nil or empty.
+	LogRecords []LogRecord
+
+	// BulkLogData is DEPRECATED.
 	BulkLogData []LogData
 }
+
+// LogData is DEPRECATED
+type LogData struct {
+	Timestamp time.Time
+	Event     string
+	Payload   interface{}
+}
+
+// ToLogRecord converts a deprecated LogData to a non-deprecated LogRecord
+func (ld *LogData) ToLogRecord() LogRecord {
```

Comment:
Not convinced without benchmark, but ok, doesn't matter. 

I once benchmarked passing `time.Time` (two uint64s) by value and by ptr, and by-value ns/op was about 60% of the by-pointer ns/op. But here the struct is 2x the size of time.Time, so meh.